src/components/DonatorForm/index.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+import { Container, Form, Button, Col, Row } from "react-bootstrap";
+import db from "../../firebaseConfig";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+
+export default function DonatorForm() {
+  const [donatorPostData, setDonatorPostData] = useState({});
+
+  const handlePostChange = (e, key) => {
+    console.log(e.target.value);
+    setDonatorPostData({ ...donatorPostData, [key]: e.target.value });
+    console.log(donatorPostData);
+  };
+  const { t } = useTranslation();
+
+  const addDonatorPost = (e) => {
+    e.preventDefault();
+    console.log("yeeyyyyyy");
```

Comment:
Can you remove the console.log from here

Reply:
PTAL